pkg/cuedefs/v1/function.cue

```diff
@@ -88,6 +88,10 @@ package v1
 		major?: uint
 		minor?: uint
 	}
+
+	retries?: {
+		attempts?: int & >=0 & <=20
```

Comment:
Should we add the default here to function definitions?

```
retries?: {
  attempts?: int & >=0 & <=20 | *3
}
```

Reply:
@tonyhb My thought for omitting was so that we can always control/change the default attempt count.

If we later increased/decreased the default, do you think users would want to use our new default if they haven't defined one? Or would they perhaps have assumed it would always be 3?